.github/workflows/type-check.yml

```diff
@@ -20,10 +20,10 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
```

Comment:
i like to run checks on the "middle" python version. for typing however i'm not sure if this is best practice or not!